homeassistant/components/home_connect/actions.py

```diff
@@ -0,0 +1,574 @@
+"""Custom actions (previously known as services) for the Home Connect integration."""
```

Comment:
I prefer using the module name `services.py`. That's still what we do in almost all integrations and we still use `services.yaml` for service action descriptions.

Reply:
Oh, I used that name because I found it at Music Assistant integration.
I guess sometime in the future all the files to actions.py will be renamed to actions.py or similar, right?

Reply:
No, the decision when the user facing names were renamed was that we won't change any API names.

Reply:
Okey then